{"uuid":"81d934a3-3a07-4bd2-b6a8-bc2a226f2aed","historyId":"d05e59a1299fbffefbcabe308ad6a1cb","fullName":"com.chiragrathava.tests.porm.TestVWOLogin_03_PropertyReader_DriverManager_POM.test_negative_vwo_login","labels":[{"name":"package","value":"com.chiragrathava.tests.porm.TestVWOLogin_03_PropertyReader_DriverManager_POM"},{"name":"testClass","value":"com.chiragrathava.tests.porm.TestVWOLogin_03_PropertyReader_DriverManager_POM"},{"name":"testMethod","value":"test_negative_vwo_login"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Selenium_AdvanceFramework"},{"name":"subSuite","value":"com.chiragrathava.tests.porm.TestVWOLogin_03_PropertyReader_DriverManager_POM"},{"name":"host","value":"LAPTOP-S9T0TVOA"},{"name":"thread","value":"23548@LAPTOP-S9T0TVOA.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"test_negative_vwo_login","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"\r\nExpecting not blank but was: \"\"","trace":"java.lang.AssertionError: \r\nExpecting not blank but was: \"\"\r\n\tat com.chiragrathava.tests.porm.TestVWOLogin_03_PropertyReader_DriverManager_POM.test_negative_vwo_login(TestVWOLogin_03_PropertyReader_DriverManager_POM.java:29)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"TC#1 - Verify that with invalid email, pass, error message is shown on the app.vwo.com","steps":[],"attachments":[],"parameters":[],"start":1751351733771,"stop":1751351751029}